pkg/kubelet/kubelet.go

```diff
@@ -1900,7 +1919,11 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	}
 
 	// Wait for volumes to attach/mount
-	if err := kl.volumeManager.WaitForAttachAndMount(pod); err != nil {
+	defaultedPod, _, err := kl.defaultPodLimitsForDownwardApi(pod, nil)
```

Comment:
Wouldn't this error still in the "standalone" case? It would seem that `LastNodeUpdateObject` will still be empty as initialNodeStatus() is called from registerWithAPIServer()